added  code for making  the text in the experiment list  responsive

Sanket2055 opened the pull request:
to make the text (list of experiment ) responsive i have used  material ui as it was already used in the project , it works really fine . adding  links to screenshot of before and after  check the screenshot here : 
 [previous text size ](https://ibb.co/q0Zwybs) 
 [New text size ](https://ibb.co/QbXx7T8)
if u think this PR is appropriate please merge it (give label hacktoberfest accepted ) before 30th october
Thank you!

TheNinza commented:
Hey @Sanket2055, the code is not working as you can see in above deployment. Also instead of customising the theme, I will prefer adding mediaQueries in the https://github.com/TheNinza/dsa-visualizer/blob/main/src/pages/experiment-list/experiment-list.styles.jsx file

Sanket2055 commented:
@TheNinza  sir  can you please review it now  , I have removed the previously created theme and added breakpoints in experiment-list.styles.jsx , now it is working fine . 
here are some screen shot's 
[widht-690](https://ibb.co/ncq6rtx)
[widht-480](https://ibb.co/wLgt7Q1)
[width-300](https://ibb.co/vLp2PZg)

TheNinza reviewed (approved):
looks good to me. Merging
